crates/nu-parser/src/parse_keywords.rs

```diff
@@ -3742,7 +3742,7 @@ pub fn parse_register(working_set: &mut StateWorkingSet, lite_command: &LiteComm
 
                 let metadata_and_signatures = plugin
                     .clone()
-                    .get(get_envs)
+                    .get(get_envs, None)
```

Comment:
and I think this is one of the problems with providing it as an argument, because now this plugin has spawned without any ctrlc handlers. I think it would be best to make sure they all get it at construction time.